packages/shared/src/components/profile/UpvoterListPlaceholder.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { ReactElement, memo } from 'react';
+import { PlaceholderList } from '../cards/PlaceholderList';
+
+export interface UpvoterListPlaceholderProps {
+  placeholderAmount?: number;
+}
+
+export function UpvoterListPlaceholder({
+  placeholderAmount = 5,
+}: UpvoterListPlaceholderProps): ReactElement {
+  return (
+    <div className="flex flex-col">
+      {Array(placeholderAmount)
+        .fill(0)
+        .map(() => (
+          <PlaceholderList key={Math.random()} />
```

Comment:
Not a good idea to use random here. You can use the map function index parameter and just disable eslint for this line

Reply:
By the way, does `PlaceholderList` look like the placeholder we need to design here? I'm not sure

Reply:
Alright, I'll disable the eslint in this case. For the usage of key, is it just on this rare case that we will disable this rule or we can do it on the config as a whole?

Out of the two placeholders, this was the one looking more like it, though not as perfectly the same from the design. Should we create another placeholder component for this?
<img width="569" alt="Screen Shot 2021-09-23 at 11 27 08 PM" src="https://user-images.githubusercontent.com/13744167/134540943-160fb83e-b9ab-47db-a308-f2415de9c78f.png">